source/api/content-state/0.1/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "IIIF Import to Viewers API 0.1"
```

Comment:
Since 0.1 was never actually published, could we rename it to "Content State"  to avoid the confusion about the history?

Reply:
see 0.2